core/src/main/java/org/apache/brooklyn/core/plan/PlanToSpecFactory.java

```diff
@@ -36,12 +40,30 @@
 import com.google.common.base.Function;
 import com.google.common.collect.ImmutableList;
 
+/** @deprecated since 0.9.0 use {@link TypePlanTransformers} as part of switch to {@link BrooklynTypePlanTransformer};
+ * mark transformers as deprecated if there is a preferred corresponding {@link BrooklynTypePlanTransformer} */
+@Deprecated 
 public class PlanToSpecFactory {
     
     private static final Logger log = LoggerFactory.getLogger(PlanToSpecFactory.class);
 
-    private static Collection<PlanToSpecTransformer> getAll() {
-        return ImmutableList.copyOf(ServiceLoader.load(PlanToSpecTransformer.class));
+    private static Collection<PlanToSpecTransformer> getAll(boolean includeDeprecated) {
```

Comment:
The `includeDeprecated` filter doesn't have good vibes :). It's all in deprecated functionality so no big deal, but still hope it doesn't stick for too long.